tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
```

Comment:
@Ivoz I was going to add some meaningful tests to confirm the data files are actually installed correctly, so I'd like to keep the tests for that reason.  as for docs, yea, sphinx is the clear default, but I'm sure as soon as we posted some sample,  some docs people would show up with many things to say about how wrong the sample was.

@pfmoore not sure where it's reduced?  other than clarifying that it never had a larger scope, and trimming the gitignore